src/DataStructures/Tensor/Expressions/Evaluate.hpp

```diff
@@ -260,6 +237,68 @@ void evaluate(
     }
   }
 }
+}  // namespace detail
+
+/*!
+ * \ingroup TensorExpressionsGroup
+ * \brief Evaluate a RHS tensor expression to a tensor with the LHS index order
```

Comment:
Sorry, I'm having a hard time understanding this brief. Maybe replace `Evaluate` with `Assign` would clarify it. I think the part I don't understand is what it means to "evaluate X to Y" but I do understand what it means to "assign X to Y"

Reply:
Addressed by new fixups https://github.com/sxs-collaboration/spectre/pull/3842/commits/f4a8355967b0d42817858a11d8927ac71025017c and https://github.com/sxs-collaboration/spectre/pull/3842/commits/05e5f1095b93e4452002d87502e5435d5504caf2